presto-docs/src/main/sphinx/release/release-0.268.rst

```diff
@@ -0,0 +1,31 @@
+=============
+Release 0.268
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add support to use consistent hashing as a hash strategy when affinity scheduling is used. This feature can be turned on by setting ``node-scheduler.node-selection-hash-strategy`` to ``CONSISTENT``. A separate configuration ``node-scheduler.consistent-hashing-min-virtual-node-count`` specifies the minimal number of virtual nodes in the consistent hashing ring.
+* Support Hive scalar function.
+* Adding limit param support for /v1/query endpoint for Multi coordinator
```

Comment:
I think we should remove this until we add documentation for multi coordinator and it's officially GA.